app/MMB/src/scenes/tripServices/LocationPopupButton.js

```diff
@@ -30,18 +32,26 @@ class LocationPopupButton extends React.Component<Props> {
   render = () => {
     const cityName = idx(this.props, _ => _.data.city.name);
     const locationId = idx(this.props, _ => _.data.locationId);
+    const date = idx(this.props, _ => _.date);
 
     return (
       <Touchable onPress={this.openWhitelabel} style={styleSheet.wrapper}>
         <React.Fragment>
           <CountryFlag data={this.props.data} />
-          {this.props.displayIata ? (
-            <Translation
-              passThrough={`in ${cityName || ''} (${locationId || ''})`}
-            />
-          ) : (
-            <Translation passThrough={cityName || ''} />
-          )}
+          <View style={styleSheet.locationWrapper}>
+            {this.props.displayIata ? (
+              <Translation
+                passThrough={`in ${cityName || ''} (${locationId || ''})`}
+              />
+            ) : (
+              <Translation passThrough={cityName || ''} />
+            )}
+            {this.props.date !== null && (
```

Comment:
this will now render if date is undefined, which one would not expect. 

change it to `this.props.date !== null`